Update the Azure Event Hub configuration (#9152)
grafana/loki: docs/sources/clients/promtail/scraping.md

```

## Azure Event Hubs
Promtail supports reading messages from Azure Event Hubs. Azure Event Hubs provides a Kafka endpoint (supports Apache Kafka protocol 1.0 and later). Azure Event Hubs exposes the same Kafka Connect API enabling the use of the Promtail Kafka Consumer with the Event Hubs service.

Targets can be configured using the `kafka` stanza:

```yaml
  - job_name: azure_event_hubs
    kafka:
      brokers:
      - my-namespace.servicebus.windows.net:9093
      topics:
      - my-event-hub-name
      group_id: my-group-id
      version: 1.0.0
      authentication:
        type: sasl
        sasl_config:
          mechanism: PLAIN
          user: '$ConnectionString'
          password: "my-connection-string"
          use_tls: true
      labels:
        job: azure_event_hub
```

> Only `broker`,`topics`, `connection_string` and `password` are required fields. Read this documentation on how to get the Azure Event Hubs connection string (https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string ) 

Reviewer: ```suggestion
Only `broker`,`topics`, `connection_string` and `password` are required fields. For information on how to get the Azure Event Hubs connection string, see the [Azure Event Hubs documentation](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string ) 
```
